apps/common/forms.py

```diff
@@ -42,6 +42,26 @@ class Meta(FlaskForm.Meta):
 AGE_VALUES = ("0-15", "16-25", "26-35", "36-45", "46-55", "56-65", "66+")
 AGE_CHOICES = tuple(OPT_OUT + [(v, v) for v in AGE_VALUES])
 
+SEXUALITY_VALUES = ("straight-or-heterosexual","gay-or-lesbian","bisexual","other")
```

Comment:
Can we do `bi-or-pansexual` here? ("bi" as in "more than one specific gender", "pan" as in "don't care about gender")